src/endpoint/installedapps.ts

```diff
@@ -189,6 +189,7 @@ export interface InstalledApp {
 	 * installed once within a user's account.
 	 */
 	singleInstance: boolean
+	restrictionTier?: number
```

Comment:
I think we should add a comment here with more detail about what this field is/does. If not here, at least somewhere in the SDK docs.

Reply:
I've been looking for someone with the answer to this question. The current swagger docs say only _"Restriction tier of the install, if any"_, which is not particularly illuminating. 

I added it here because someone from the Coda team suggested it might have something to do with the recent change that does not show certain installed apps OneApp is not in dev mode. I'm not sure that's the case though. I think it has something to do with the restrictions on devices like Nest. 

For now though, I think that representing it in the typescript is preferable to leaving it out and still returning it in the response.

Reply:
We should at least add a `TODO` comment to add more info. Seems more likely to happen at some point then.